modules/ROOT/pages/dw-arrays-functions-countby.adoc

```diff
@@ -51,5 +51,5 @@ output application/json
 
 Helper function that enables `countBy` to work with a `null` value.
 
-_Introduced in DataWeave: 2.4.0._
+_Introduced in DataWeave version 2.4.0._
```

Comment:
TODO (DEV): _Introduced in DataWeave: -> _Introduced in DataWeave version

fix all instances of this